packages/nimble-components/src/table/index.ts

```diff
@@ -609,6 +629,23 @@ export class Table<
         this.tableUpdateTracker.trackColumnInstancesChanged();
     }
 
+    private getCurrentSelectedRecordIds(): string[] {
```

Comment:
nit: maybe this logic could be moved to selection manager base as a public api method?
selection manager has the tanstack table reference which seems to be able to lookup the same thing, i.e. tanstackTable.options.state.rowSelection